src/modules/storage.js

```diff
@@ -0,0 +1,15 @@
+const storageKey = 'todolist';
+
+const saveTasks = (arr) => {
+  localStorage.setItem(storageKey, JSON.stringify(arr));
+};
+
+const loadTasks = () => {
+  let arr = JSON.parse(localStorage.getItem(storageKey));
+  if (arr == null) {
```

Comment:
- You should you instead of '==' use strict equal sign '==='

Reply:
Thanks!